### DISCUSSION TITLE
festivetech theme days till christmas

### DISCUSSION THREAD

**Discussion by tapatiohaxx** [JanDeDobbeleer/oh-my-posh#2478] [Troubleshoot] [answered]
I am new to doing stuff with Windows system commands, can anyone here explain to me how to remove the tree icon and get  the correct amount of days until Christmas?
![Screenshot (2)](https://user-images.githubusercontent.com/33331153/177018359-1032252c-b60a-435f-bc24-e88c7a1745b7.png)

**Comment by lewis-yeung** [comment, marked as answer]
@tapatiohaxx In fact, this is not handled by OMP itself. Related code:

https://github.com/JanDeDobbeleer/oh-my-posh/blob/9f9b91e56688ed10014666a4ca815a50e794bfb9/themes/festivetech.omp.json#L69

And please see [this](https://github.com/JanDeDobbeleer/oh-my-posh/pull/1326#issue-1064841785): 

> this theme is a bit special and needs the following in your
PowerShell profile to work correctly:
>
> function Set-EnvVars {
$now = [DateTime]::Now
$env:DAYS = ([Datetime](“12/25/” + $now.Year) – $Now).Days
}
New-Alias -Name 'Set-PoshContext' -Value 'Set-EnvVars' -Scope Global -Force

As we can see, we need a trick to make the env `DAYS` reflect the amount of days until Christmas. :)

**Reply by tapatiohaxx** [reply]
Thank you very much for linking me to this code. I am happy to have this theme working correctly. 
Hope you have a nice day!

**Reply by KamilaMattos** [reply]
@tapatiohaxx did you make it work? where exactly do i need to put the mentioned code?  sorry i'm newbie...

**Reply by lewis-yeung** [reply]
@KamilaMattos You should put the following code in your [PowerShell profile](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_profiles?view=powershell-7.2) to make it work:

```powershell
function Set-EnvVars {
  $now = [DateTime]::Now
  $env:DAYS = ([Datetime]("12/25/" + $now.Year) - $now).Days
}
New-Alias -Name 'Set-PoshContext' -Value 'Set-EnvVars' -Scope Global -Force
```

**Reply by tapatiohaxx** [reply]
No worries, what I did to solve this problem is created two files, one  at `C:\Users\[My_USER]\Documents\PowerShell\Microsoft.PowerShell_profile.ps1` and one at `C:\Users\[My_USER]\.config\powershell\user_profile.ps1`. Once I did that, I used the files in my repo at ps-user-scripts to fill each of the above files. 

The main `$PROFILE` serves as a pointer to the user profile which contains all of the cool theming, aliases, environment variables, and other things you want to set your environment up with. 

Remember to change the paths to the theme files you have on your machine for everything to work well.

Hope it all works out for you!

**Reply by KamilaMattos** [reply]
@lewis-yeung  and @tapatiohaxx thank you so much guys!! 🚀
